index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="header">
```

Comment:
- [ ] Please for semantic reasons I would recommend putting your hamburger menu and your logo inside a `nav` tag and wrapping them with a `header` tag.
- [ ] Your `logo` can just be plain text inside an `a` tag instead an image.
- [ ] Your `hamburger` menu is different compared to the one in the original design. Please make sure to fix that by using the same hamburger menu and wrapping it inside a `button`.

### After the change is done, you can have something like this 👇️

```
<header class="header">
      <nav>
        <a class="logo">My Logo</a> 
        <button class="hamburger"><img src="./img/hamburger.png" alt=""></button>
      </nav>
</header>

```

Reply:
Hi @dazehinn! The hamburger is the navigation, you're right, but for the browser, it's just an "Image". That's why we use the `<nav>` tag, to tell the browser that this is part of the navigation of the page. 🤓 This is especially useful for assistive technologies such as screen readers for people with visual impairment, improving a lot the accessibility of your page, which is a concept you're going to learn later in the program! 🚀